readmes/README.Linux.md

```diff
@@ -1,20 +1,28 @@
-# Set up development environment for Cloudberry Database on Linux
+# Compile and Install Cloudberry Database on Linux
 
-This document shares how to set up the development environments on different Linux systems (CentOS 7, RHEL, and Ubuntu). Before you start to [explore the Cloudberry Database](https://github.com/cloudberrydb/cloudberrydb/tree/main/readmes), follow this document to download and install the required tools and dependencies, and create demo user.
+This document shares how to compile and install Cloudberry Database on Linux systems (CentOS 7, RHEL, and Ubuntu). Note that this document is for developers to try out Cloudberry Database in a single-node environments. DO NOT use this document for production environments.
+
+For this guide to work properly, you need to ensure that your (virtual) machine has at least 4-core CPU and 8 GB memory.
```

Comment:
We can delete the machine requirement here. I tested on one machine with 2core, 4G memory, it can work as well.